components/Dialogs/src/MDCDialogTransitionController.m

```diff
@@ -18,13 +18,24 @@
 
 @implementation MDCDialogTransitionController
 
-static const NSTimeInterval MDCDialogTransitionDuration = 0.27;
+static const NSTimeInterval MDCDialogDefaultTransitionDuration = 0.27;
```

Comment:
Nit: Please [document this constant and the one below](http://google.github.io/styleguide/objcguide.html#declaration-comments).